hack/tests/check-lint.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -e
+
+source ./hack/common.sh
+
+CI_LINTERS=(
+    "--enable=nakedret"
+    "--enable=goconst"
+)
+
+DEV_LINTERS=(
+    ##todo(camilamacedo86): The following checks requires fixes in the code.
+    ##todo(camilamacedo86): they should be enabled and added in the CI
+    "--enable=golint"
+    "--enable=gocyclo"
+    "--enable=goimports"
+    "--enable=lll"
+    "--enable=goconst"
+    "--enable=gosec"
+    "--enable=maligned"
+    "--enable=deadcode"
+    "--enable=misspell"
+    "--enable=errcheck"
+    "--enable=dupl"
+    "--enable=interfacer"
+    "--enable=ineffassign"
+    "--enable=varcheck"
+    "--enable=structcheck"
+    "--enable=unparam"
+)
+
+subcommand=$1
+case $subcommand in
+	"dev")
+	  ##todo(camilamacedo86): It should be removed when all linter checks be enabled
+	  header_text "Checking the project with all linters (dev)"
+		LINTERS=${DEV_LINTERS[@]}
+		;;
+	"ci")
+	  header_text "Checking the project with the linters enabled for the ci"
+		LINTERS=${CI_LINTERS[@]}
+		;;
+	*)
+		echo "Must pass 'dev' or 'ci' argument"
+		exit 1
+esac
+
+fetch_go_linter
+
+header_text "Running golangci-lint"
+golangci-lint run --disable-all \
+    --deadline 5m \
+    --enable=nakedret \
+    --enable=goconst \
```

Comment:
Use `${LINTERS[@]}` here.

Reply:
good catch. I removed by mistake in the last changes. Done